crypto-ffi/src/wasm/mod.rs

```diff
@@ -75,7 +75,9 @@ pub struct BuildMetadata {
 // This is intended to hotfix this import:
 // ❯ wasmer inspect bindings/js/wasm/core-crypto-ffi_bg.wasm | grep env
 //    "env"."__stack_chk_fail": [] -> []
-#[no_mangle]
+//
+// SAFETY: This symbol is globally unique.
+#[unsafe(no_mangle)]
 pub extern "C" fn __stack_chk_fail() {
     panic!("Stack overflow detected");
 }
```

Comment:
This makes sense only if stack protector is enabled, which it isn't in CC. So I think we should just remove this.

Reply:
I don't mind in principle, but that's out of scope for this PR. I don't know where stack protection is or was configured, and I'd really rather avoid another 0.5-1 hours spent removing this and waiting for CI again, especially if it causes some test to fail.